README.md

```diff
@@ -1,32 +1,28 @@
-## Go Ethereum
+## Go Quai
 
-Official Golang implementation of the Ethereum protocol.
+Official Golang implementation of the Quai protocol.
 
 [![API Reference](
 https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/68747470733a2f2f676f646f632e6f72672f6769746875622e636f6d2f676f6c616e672f6764646f3f7374617475732e737667
-)](https://pkg.go.dev/github.com/ethereum/go-ethereum?tab=doc)
-[![Go Report Card](https://goreportcard.com/badge/github.com/ethereum/go-ethereum)](https://goreportcard.com/report/github.com/ethereum/go-ethereum)
+)](https://pkg.go.dev/github.com/dominant-strategies/go-quai/common)
+[![Go Report Card](https://goreportcard.com/badge/github.com/spruce-solutions/go-quai)](https://goreportcard.com/report/github.com/spruce-solutions/go-quai)
```

Comment:
these need to be updated from spruce-solutions to dominant-strategies

Reply:
https://goreportcard.com/report/github.com/dominant-strategies/go-quai